packages/material-ui-lab/src/Autocomplete/Autocomplete.d.ts

```diff
@@ -101,6 +101,14 @@ export interface AutocompleteProps
    * @returns {ReactNode}
    */
   renderTags?: (value: any, state: RenderValueState) => React.ReactNode;
+  /**
+   * Object of titles to display when hovering the arrow or clear buttons.
```

Comment:
No need for the prop type in the description.

Reply:
You mean the `Object` part?

Reply:
He does, it should probably just start with “Titles to display...”

Reply:
Thanks Josh.